proto/cosmos/staking/v1beta1/tx.proto

```diff
@@ -35,6 +35,8 @@ service Msg {
 
   // CancelUnbondingDelegation defines a method for performing canceling the unbonding delegation
   // and delegate back to previous validator.
+  //
+  // Since: cosmos-sdk 0.46
```

Comment:
Any ideas how we can enforce this better, that we don't forget then in normal PRs?